ISSUE_TEMPLATE.md

```diff
@@ -4,13 +4,20 @@
 - Discussion Link: <!-- link to SIG mailing list thread, meeting, or recording where the Enhancement was discussed before KEP creation -->
 - Primary contact (assignee):
 - Responsible SIGs:
-- Enhancement target (which target equals to which milestone):
```

Comment:
I would leave this here since I think an at a glance target list is good..

Reply:
Thanks for the feedback, it's been updated! :D